packages/truffle-box/test/utils-index.js

```diff
@@ -0,0 +1,13 @@
+const assert = require("assert");
+const utils = require("../lib/utils");
+
+describe("utils", () => {
+  it("setUpBox throws when passed an invalid boxConfig", () => {
+    let boxConfig = {};
+
+    utils
+      .setUpBox(boxConfig)
+      .then(() => assert(false, "didn't throw!"))
+      .catch(e => assert(e.stack.includes("Error")));
```

Comment:
You might want to test against this error a little more strictly to ensure that the type of error you expect is being thrown. Currently, if this test throws for probably ANY reason, it will pass. This could lead to false positives.